samples/README.md

```diff
@@ -2,324 +2,215 @@
 [//]: # "To regenerate it, use `npm run generate-scaffolding`."
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# Google Cloud Storage: Node.js Samples
+# [Google Cloud Storage: Node.js Samples](https://github.com/googleapis/nodejs-storage)
 
 [![Open in Cloud Shell][shell_img]][shell_link]
 
-[Cloud Storage](https://cloud.google.com/storage/docs) allows world-wide storage and retrieval of any amount of data at any time. You can use Google Cloud Storage for a range of scenarios including serving website content, storing data for archival and disaster recovery, or distributing large data objects to users via direct download.
+> Node.js idiomatic client for [Cloud Storage][product-docs].
```

Comment:
I don't think this is rendering correctly:
![image](https://user-images.githubusercontent.com/534619/55120233-51e6ed00-50b2-11e9-9ab6-348021883c68.png)

Reply:
this should be fixed now :+1: